components/Header.tsx

```diff
@@ -12,7 +12,7 @@ export default function Header({ query = '' }: Props) {
         <div className="flex h-20 py-4 px-5 md:px-8 justify-center items-center">
           <div className="flex flex-1">
             <Link
-              href="/record"
+              href="/"
```

Comment:
I wonder if this should trigger the recording to start once we land on the home page? Maybe with a query param?